reference/5.1/Microsoft.PowerShell.Core/About/about_PSCustomObject.md

```diff
@@ -94,6 +95,20 @@ one two
   1   2
 ```
 
+**PSObject** type objects maintain the list of members in the order that the
+members were added to the object. **Hashtable** objects don't guarantee the
+order of the key-value pairs. Therefore, if you want properties of the
+**PSObject** to appear in a specific order you must add them using `Add-Member`
+or use and ordered hashtable. For example:
+
+```powershell
+$Asset = [pscustomobject]([ordered]@{
```

Comment:
I know it's late but you definitely don't need `[ordered]` if you are doing `[PSCustomObject]@{...}` in line. The ordered is only needed if you are building the hashtable separately. If the latter is what this is trying to convery then the example should be updated to store the hashtable in a separate var and then caseted rather than what this shows. For example this is perfectly fine and will preserve the order as written

```powershell
$Asset = [PSCustomObject]@{
    Name       = "Server30"
    System     = "Server Core"
    PSVersion = "4.0"
}
```